app/grandchallenge/components/models.py

```diff
@@ -662,32 +662,68 @@ class OverlaySegmentsMixin(models.Model):
 
     @property
     def voxel_values(self):
-        allowed_values = {x["voxel_value"] for x in self.overlay_segments}
+        return {x["voxel_value"] for x in self.overlay_segments}
 
+    @property
+    def allowed_values(self):
+        allowed_values = self.voxel_values
         # An implicit background value of 0 is always allowed, this saves the
         # user having to declare it and the annotator mark it
         allowed_values.add(0)
 
         return allowed_values
 
+    @property
+    def is_contiguous(self):
+        values = sorted(list(self.voxel_values))
```

Comment:
The implicit background value does not need to be included in this check?

Reply:
I wasn't sure about that. It depends on what we allow. For now I left it out so that it has to be explicitly defined. I'm not really sure how this is supposed to work with signed integer datatypes either.

Reply:
We could drop the contiguous checks now - the TIFF file will now fill out the segments with all the values between min an max and the existing checks will catch if there is a problem. Telling a user that the interface must be contiguous isn't going to be helpful. If we find a better way to determine the segments for the TIFF file in panimg then all this will still work too.

However, it would be helpful that there is a check for contiguous segments for new interface definitions so that we eliminate this corner case for now.

Reply:
I've removed the contiguous check in the voxel value validation and added a check to `_clean_overlay_segments` so new and existing component interfaces cannot be saved when the overlay segments are not contiguous.